sdk/purview/Azure.Analytics.Purview.Sharing/samples/Sample10_ListDetachedReceivedShares.md

```diff
@@ -0,0 +1,20 @@
+# Azure Share Client Samples - List Detached Received Shares
+
+## Import the namespaces
+
+```C# Snippet:ReceivedSharesClientSample_ImportNamespaces
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Identity;
+```
+
+## List Detached Received Shares
+
+```C# Snippet:ReceivedSharesClientSample_ListDetachedReceivedShares
```

Comment:
We typically include a short paragraph to describe the scenario where I, as a user of the service, would want to do this in my application.